apps/client/src/app/[user]/components/RecommendSection/RecommendList/index.tsx

```diff
@@ -0,0 +1,32 @@
+import SearchEmpty from "@/app/[user]/components/RecommendSection/SearchEmpty";
+import RecommendCard from "@/app/[user]/components/RecommendSection/RecommendCard";
+import { studyListWrapper } from "@/app/[user]/components/RecommendSection/RecommendList/index.css";
+import type { Study } from "@/app/api/groups/type";
+
+type RecommendListProps = {
+  studyList: Study[];
+};
+
+const RecommendList = ({ studyList }: RecommendListProps) => {
+  return (
+    <>
+      {studyList && studyList.length > 0 ? (
+        <ul className={studyListWrapper}>
+          {studyList.map((study) => (
+            <li key={study.id}>
+              <RecommendCard
+                name={study.name}
+                introduction={study.introduction}
+                groupImage={study.groupImage}
+              />
+            </li>
+          ))}
+        </ul>
+      ) : (
+        <SearchEmpty />
+      )}
+    </>
+  );
+};
```

Comment:
```suggestion
const RecommendList = ({ studyList }: RecommendListProps) => {

  if (!(studyList && studyList.length > 0)) {
    <SearchEmpty />
  }
  
  return (
        <ul className={studyListWrapper}>
          {studyList.map((study) => (
            <li key={study.id}>
              <RecommendCard
                name={study.name}
                introduction={study.introduction}
                groupImage={study.groupImage}
              />
            </li>
          ))}
        </ul>
};
```

미리 early return 시키는 방향은 어떠신가용?
내부 세부적인 사항을 삼항 연산자로 조건 나누는게 아니라, 컴포넌트 전체가 어떤 조건에 의해 아예 다른 컴포넌트를 반환시켜야한다면,
early return 시켜주는 것이 가독성 측면에서 더 좋다고 생각합니다!

[관련 아티클](https://medium.com/swlh/return-early-pattern-3d18a41bba8)

Reply:
요거는 저번 코멘트에서 말씀드렸던 내용인데 제가 체크를 못했네요. 아티클 내용 참 좋으니 한번 읽어보시는거 추천해요!